src/main/java/lotto/model/MatchNumber.java

```diff
@@ -0,0 +1,41 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+
+public enum MatchNumber {
+
+
+    ZERO(0, 0),
+    ONE(1, 0),
+    TWO(2, 0),
+    THREE(3, 5000),
+    FOUR(4, 50000),
```

Comment:
```suggestion
    FOUR(4, 50_000),
```

언더스코어(_) 를 활용하면 가독성을 좀 더 올릴 수 있습니다!